FND/DatesAndTimes/Occurrences.rdf

```diff
@@ -257,19 +257,20 @@ They are modularized this way to minimize the ontological committments that are
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-dt-oc;isInputTo">
 		<rdfs:label>is input to</rdfs:label>
-		<skos:definition>indicates an input to some activity, process, system, report, analysis, etc.</skos:definition>
+		<skos:definition>indicates a precondition, requirement, or other contribution (e.g., data) to some activity, process, system, report, analysis, etc.</skos:definition>
```

Comment:
This is an example of why we need the new hygiene test. This whole semantic seems to be not well-formed.